op-challenger/game/fault/solver/solver_test.go

```diff
@@ -32,9 +34,9 @@ func TestAttemptStep(t *testing.T) {
 		{
 			name:               "AttackFirstTraceIndex",
 			expectAttack:       true,
-			expectPreState:     claimBuilder.CorrectPreState(0),
-			expectProofData:    claimBuilder.CorrectProofData(0),
-			expectedOracleData: claimBuilder.CorrectOracleData(0),
+			expectPreState:     claimBuilder.CorrectPreState(new(big.Int)),
+			expectProofData:    claimBuilder.CorrectProofData(new(big.Int)),
+			expectedOracleData: claimBuilder.CorrectOracleData(new(big.Int)),
```

Comment:
These would more more typical as `big.NewInt(0)` or `common.Big0`.  😄  The `new(big.Int)` is typically just when it's being used as the receiver of some computation (ie `new(big.Int).Add(big.NewInt(0), big.NewInt(1))`.